.ci/travis.sh

```diff
@@ -138,6 +138,10 @@ jacoco)
   fi
   ;;
 
+quarterly-cache-cleanup)
+  find ~/.m2 -mtime +90 -exec rm {} \;
+  ;;
```

Comment:
It would be awesome if we could base cache cleaning on changed SHA of pom.xml

Reply:
We changing versions too often in pom.xml

Reply:
Also pom can change for other things besides version upgrades.

I will come back to this PR to unblock it.

Reply:
I do not recommend to block this PR, we can always improve later on.